gotocompany/optimus/core/v1beta1/resource.proto

```diff
@@ -56,6 +56,13 @@ service ResourceService {
       body: "*"
     };
   }
+
+  // ChangeJobNamespace move a job spec from one namespace to another
+  rpc ChangeResourceNamespace(ChangeResourceNamespaceRequest) returns (ChangeResourceNamespaceResponse) {
+    option (google.api.http) = {
+      put: "/v1beta1/project/{project_name}/namespace/{namespace_name}/datastore/{datastore_name}/resource/{resource_name}"
```

Comment:
ideally the endpoint here could be used for a more general update operation. but, the update operation that we have (other than this change namespace) uses a different kind of endpoint. do let me know if discussion is required.

Reply:
yeah we have chosen an action specific name for the path now